### PR TITLE
Collapsible End Pane in Vertical Layout

### PR DESCRIPTION
**Pull request by wldcordeiro**
Associated Issue: #1537 

**Requires devtools-html/devtools-core#88 to land to actually work properly.**

### Summary of Changes

* add the pane collapse button to the source footer and wire up logic to toggle end pane collapse

### Test Plan

Example test plan:

- [x] Open Debugger in vertical mode
- [x] Toggle collapse button for bottom
- [x] See the pane has collapsed/uncollapsed

### Screenshots/Videos (OPTIONAL)

screenshot_20170114_205621
screenshot_20170114_205630

@jasonLaster One of the only issues I have with this approach is that the collapse button isn't visible until you open a file. I may instead render it as part of the "editorPane" in `App.js` so that it's always visible. @

### CODE REVIEW

**Comment by jasonLaster** [comment]
oh shit - that's a really interesting problem. 

Perhaps we want to render the footer in App.js so that we can always show the footer... We do this for the SourceTabs as well.

**Comment by jasonLaster** [comment]
Thanks @wldcordeiro!  

Your work looks really good. I took a stab at adding the end toggle button to the welcome box as well.

**Comment by wldcordeiro** [comment]
This looks good with the changes @jasonLaster I'm going to bring it in. 👍

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "codemirror": "^5.1.0",
-    "devtools-launchpad": "0.0.23",
+    "devtools-launchpad": "0.0.24",
```
Comment (clarkbw) [inline package.json]: We need some kind of yarn-lint that checks we've updated the `yarn.lock` file whenever the `package.json` updates.

Comment (jasonLaster) [inline package.json]: Good catch

**Pull request merged**